frontend/public/components/persistent-volume.jsx

```diff
@@ -1,23 +1,47 @@
 import * as React from 'react';
+import { Link } from 'react-router-dom';
 
 import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
-import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp } from './utils';
+import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp, StatusIcon } from './utils';
+
+const pvPhase = pv => pv.status.phase;
 
 const { common } = Kebab.factory;
 const menuActions = [...common];
 
+const PVStatus = ({pv}) => {
+  const phase = pvPhase(pv);
+  return <StatusIcon status={phase} />;
+};
+
 const Header = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.labels">Labels</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="spec.capacity.storage">Capacity</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="metadata.labels">Labels</ColHead>
```

Comment:
I don't think all of these fields will fit at mobile. We'd need to drop some columns.

Reply:
that's correct I will hide some columns from xs screen